src/components/FloatingHelper/ClosablePopover/ClosablePopover.tsx

```diff
@@ -37,28 +42,63 @@ export type ClosablePopoverProps = PickedPopoverProps & ClosablePopoverOwnProps;
  * calling a closeAction.
  */
 export class ClosablePopover extends React.PureComponent<ClosablePopoverProps, ClosablePopoverState> {
-  state: ClosablePopoverState = { opened: true };
+  state: ClosablePopoverState;
 
   static propTypes: React.ValidationMap<ClosablePopoverProps> = {
     ...pickedPopoverPropTypes,
     opened: bool,
     content: func,
-    target: node
+    target: node,
+    onOpened: func,
```

Comment:
I remember reading somewhere a guideline to when to use past tense and when present tesne.
If the event consequences (default behavior) already happened, and the listener has no way of preventing it, then use past tense. If the event has no consequences or the listener can prevent it from happening (using something like `event.preventDefault`) then use present tense.
I saw that material ui also uses this gudeline.
I also added this guideline to this [WIP document]( https://docs.google.com/document/d/1Hp98GWEY_c-96FZhefhP8UzpXGTmYeTk6cmm40x0ED4/edit#)

Material-UI uses some similar tense variation , for example in [Popover](https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/Popover/Popover.js)

For our case: I implemented it so that the callback is called after we already opened (or closed) the popover, so I used past tense. Do you think I should change the implementation?

Reply:
for me as consumer will there ever be a difference between `onOpen` & `onOpened`? it makes sense in cases like `onDrag` & `onDragged` but here it feels like over thinking.

on a side note, what are we doing here if material has everything done already?